.github/workflows/ci.yml

```diff
@@ -27,13 +27,15 @@ jobs:
         NXF_VER:
           - "23.04.0"
           - "latest-everything"
-        parameters:
-          - "--preprocessing_qc_tool falco"
-          - "--shortread_qc_tool fastp"
-          - "--shortread_qc_tool adapterremoval"
-          - "--shortread_complexityfilter_tool bbduk"
-          - "--shortread_complexityfilter_tool prinseqplusplus"
-
+        tags:
```

Comment:
We still want the basic `test` no?

Reply:
Do you mean I should only include the `test`? Initially, I included all the other tests as well, but I eventually removed them since they were not part of the original CI test.

Reply:
Sorry, I meant that every config should be included (except maybe `test_nothing` should be in the list)

Reply:
Ok